backend/src/main/resources/db/migration/V1.22__organism_index.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX IF NOT EXISTS sequence_entries_organism_idx ON public.sequence_entries USING btree (organism);
```

Comment:
```suggestion
CREATE INDEX IF NOT EXISTS sequence_entries_organism_idx ON sequence_entries (organism);

```

- We don't specify a schema (`public`).
- The default is `btree`, so we don't need to specify it.